Add Room Booking Guidelines (#52)
kossiitkgp/docs: events/room-booking.md

# Room Booking

## Steps for Room Booking

> - Go to the Junior Executive, Academic Section (UG) (Mr. Shouvik Adhikari as of now) and get the form for room booking.
> - This form is only for room booking and not for AV support.
> - Fill in the the details necessary for booking room. Date and time, Duration.
> - Would need signature of a Professor and the HOD of the respective department as the booking is made on the professors name. (The most tough part of job). CSE professors are easily approachable for this, so can reach them out.
> - Submit this form back to the Jr. Executive, and he would reach back in a day or two for the payment.
> - The Jr. Executive would give a draft to be paid in the fees counter and take the payment receipt.
> - Upon submitting the reciept to the Jr. Executive, the process from our end is done. And Voila, room booking is finally done.
> 
> ## Steps for AV cell
> 
> - Get the AV cell form from the AV cell room in nalanda , or better to get it from CWISS office near Mechanical Engginering department.
> - Rest steps to be unknown right now..
> 
> ## Points to be noted
> 
> - Keep atleast 1.5 week in hand before booking a room, the process takes time.
> - UPI payment is accepted while paying in Fees department.
> - Both the form for room booking and AV cell needs professor's signature, better to get it done together.
> 
> ## Rates of Room w.e.f Oct 5, 2022
> 
> | Sl no. | Name of the Auditorium                   | Seating Capactiy | 4hrs | 8 hrs |
> | ------ | ---------------------------------------- | ---------------- | ---- | ----- |
> | 1      | Bhatnagar (F-116) / Raman (F-142)        | 300              | 2000 | 3000  |
> | 2      | S-301/S-302                              | -                | 1500 | 2500  |
> | 3      | V1 /V2 Class Room                        | 359              | 2000 | 3000  |
> | 4      | V3/V4 Class Room                         | 179              | 1500 | 2500  |
> | 5      | Nalanda Class Room(Upto 4 room with Ac)  | 4\*120 or 240    | 5000 | 8000  |
> | 6      | Nalanda Class Room (per room without Ac) | 120              | 500  | 800   |

Reviewer: ```suggestion
- Go to the Junior Executive, Academic Section (UG) (Mr Shouvik Adhikari as of now) and get the form for room booking.
- This form is only for room booking and not for AV support.
- Fill in the details necessary for booking a room. Date and time, Duration.
- Would need the signature of a Professor and the HOD of the respective department as the booking is made in the professor's name. (The toughest part of the job). CSE professors are easily approachable for this, so can reach them out.
- Submit this form back to the Jr. Executive, and he would reach back in a day or two for the payment.
- The Jr. Executive would give a draft to be paid at the fees counter and take the payment receipt.
- Upon submitting the receipt to the Jr. Executive, the process from our end is done. And Voila, room booking is finally done.

## Steps for AV cell

- Get the AV cell form from the AV cell room in Nalanda, or better get it from the CWISS office near the Mechanical Engineering department.
- Rest steps are unknown right now.

## Points to be noted

- Keep at least 1.5 weeks in hand before booking a room, the process takes time.
- UPI payment is accepted while paying in the Fees department.
- Both the form for room booking and AV cell needs the professor's signature, better to get it done together.

## Rates of Room w.e.f Oct 5, 2022

| Sl no. | Name of the Auditorium                   | Seating Capacity | 4hrs | 8 hrs |
| ------ | ---------------------------------------- | ---------------- | ---- | ----- |
| 1      | Bhatnagar (F-116) / Raman (F-142)        | 300              | 2000 | 3000  |
| 2      | S-301/S-302                              | -                | 1500 | 2500  |
| 3      | V1 /V2 Class Room                        | 359              | 2000 | 3000  |
| 4      | V3/V4 Class Room                         | 179              | 1500 | 2500  |
| 5      | Nalanda Class Room(Upto 4 room with Ac)  | 4\*120 or 240    | 5000 | 8000  |
| 6      | Nalanda Class Room (per room without Ac) | 120              | 500  | 800   |
```

Author: Just wanted to know what does the red words in the updated changes mean? Do they represent those words are going to get deleted?

Reviewer: It's some sort of syntax highlighting, I believe.
